docs/_includes/linter-group-listing.html

```diff
@@ -36,6 +36,11 @@ <h4 class="glue-expansion-panel__button-header">
     <div class="glue-expansion-panel-content" id="panel-aip-{{ pbr.name }}">
       <div class="has-rule-listing">
         <table class="glue-table--datatable glue-table--stacked api-linter-rule-listing" style="width: 100%;">
+          <tr>
+            <td>
+              <a href="/{{ pbr.name }}">[index]</a>
```

Comment:
Should this be `p` since the loop is `for p in pbr.items`? I honestly don't know.

Reply:
We want the outer loop var, `pbr`, because this is the link to the index of all `p`.